sw/host/tests/manuf/manuf_scrap/src/main.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use structopt::StructOpt;
+
+use opentitanlib::app::TransportWrapper;
+use opentitanlib::dif::lc_ctrl::{DifLcCtrlState, LcCtrlReg};
+use opentitanlib::execute_test;
+use opentitanlib::io::jtag::{JtagParams, JtagTap};
+use opentitanlib::test_utils::init::InitializeTest;
+use opentitanlib::test_utils::lc_transition::trigger_lc_transition;
+
+#[derive(Debug, StructOpt)]
+struct Opts {
+    #[structopt(flatten)]
+    init: InitializeTest,
+
+    #[structopt(flatten)]
+    jtag_params: JtagParams,
+
+    #[structopt(
+        long, parse(try_from_str = DifLcCtrlState::parse_lc_state_str),
```

Comment:
If you implement `FromStr`, I don't think you need an explicit `parse()` stanza.

If you also implement variants, you can say something like:
```
#[structopt(
  long,
  possilbe_values = &DifLcCtrlState::variants(),
  default_value = "foo",
  help = "<whatevs>",
)]
```